WordPress/Classes/Utility/WPError.m

```diff
@@ -82,9 +83,10 @@ + (void)showNetworkingAlertWithError:(NSError *)error title:(NSString *)title
             default:
                 break;
         }
-    } else if ([error.domain isEqualToString:WordPressComApiErrorDomain]) {
-        DDLogError(@"wp.com API error: %@: %@", [error.userInfo objectForKey:WordPressComApiErrorCodeKey], [error localizedDescription]);
-        if (error.code == WordPressComApiErrorInvalidToken || error.code == WordPressComApiErrorAuthorizationRequired) {
+    } else if ([error.domain isEqualToString:WordPressComRestApiErrorDomain]) {
```

Comment:
Shouldn't we also keep the check for `WordPressComApiErrorDomain` until `WordPressComApi` is fully removed?